<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsClass</class>
 <widget class="QMainWindow" name="settingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings - Numeric Engine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="displayFormat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Display Format</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="auto_format">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Automatic (based on number size)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="round_format">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Round result to</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="scientific_format">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Display result in Scientific Notation, with</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dp_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
     </font>
    </property>
    <property name="text">
     <string>decimal places</string>
    </property>
   </widget>
   <widget class="QLabel" name="sf_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
     </font>
    </property>
    <property name="text">
     <string>significant digits</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="displayFormat_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Customization</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
